dataplane/linux/endpoint_mgr.go

```diff
@@ -1047,6 +1047,12 @@ func (m *endpointManager) configureInterface(name string) error {
 			"Skipping configuration of interface because it is oper down.")
 		return nil
 	}
+
+	err := m.writeProcSys(fmt.Sprintf("/proc/sys/net/ipv6/conf/%s/accept_ra", name), "0")
```

Comment:
I'm concerned that this will break IPv6 for an OpenStack VM. If I'm correctly understanding that this will prevent RAs getting through to the workload, I believe that means that a VM won't set up an IPv6 default route.